src/components/TodoList/TodoList.module.css

```diff
@@ -0,0 +1,73 @@
+    .container {
+        width: 200%;
+        min-width: 450px;
+        background-color: #ffffff;
+        margin: auto;
+        padding: 20px;
+        box-shadow: 2px 2px 10px rgba(22, 22, 22, 0.2);
+        transform: translateX(-25%);
+    }
+    
+    .header {
+        display: block;
+        color: #0014f8;
+        text-align: center;
+        font-size: 40px;
+        font-weight: 900;
+        font-weight: bold;
+    }
+    
+    .input {
+        width: 100%;
+        border: 2px solid #f0f0f0;
+        padding: 10px 5px;
+        font-size: 20px;
+    }
+    
+    hr {
+        background-color: #f0f0f0;
+        border: 1px solid #f0f0f0;
+        margin-top: 2rem;
+    }
+    
+    #listContainer {
+        padding-left: 0;
+    }
+    
+    #listContainer li {
+        list-style-type: none;
+        text-align: left;
+        position: relative;
+        font-size: 1rem;
+        padding: 16px 16px 16px 0;
+        border-bottom: 1px solid black;
+    }
+    
+    #listContainer li input[type="checkbox"] {
+        width: max-content;
+        vertical-align: sub;
+        cursor: pointer;
+    }
+    
+    #listContainer li span {
+        vertical-align: middle;
+    }
+    
+    .edit {
+        position: absolute;
+        right: 50px;
+        top: 20px;
+        cursor: pointer;
+    }
+    
+    .del {
+        position: absolute;
+        right: 20px;
+        top: 20px;
+        cursor: pointer;
+    }
```

Comment:
Here you can have a common class that has the common css and del, edit will only have 'right' value.
And why are we using absolute positioning instead of flex or grid?